web/templates/projects/project.html

```diff
@@ -65,6 +65,28 @@ <h1>{{ project.acronym }} </h1>
                     </dl>
                 </div>
                 <div class="floating-buttons-absolute">
+                    {% if request.user.is_superuser %}
+                        <p class="text-center mt-5">
+                            {% if not project.is_published %}
+                                <a href="{% url 'project_publish' pk=project.pk %}" 
+                                   id="publish" 
+                                   title="Publish project"
+                                   class="btn btn-primary bmd-btn-fab float-right">
+                                    <i class="material-icons">publish</i>
+                                </a>
+                            {% else %}
+                                {% comment %}
```

Comment:
Just to confirm - this doesn't have to be a comment any more, right? The views are just redirecting without actually doing anything :+1: